datastore/store.go

```diff
@@ -237,7 +237,10 @@ func (db *CachedDB) GetEpochWeight(epoch types.EpochID) (uint64, []types.ATXID,
 }
 
 // IterateEpochATXHeaders iterates over ActivationTxs that target an epoch.
-func (db *CachedDB) IterateEpochATXHeaders(epoch types.EpochID, iter func(*types.ActivationTxHeader) error) error {
+func (db *CachedDB) IterateEpochATXHeaders(
+	epoch types.EpochID,
+	iter func(*types.ActivationTxHeader) error,
+) error {
```

Comment:
Did you change your formatting settings? I don't mind lines that are 120 (or even 160) characters long. Limiting ourselves to 80 characters seems archaic to me.

Reply:
will merge it as is (because i cherrypicked it to v1.2.2) and update config for next patches

Reply:
@dshulyak I prefer up to 120. Longer lines don't fit in side-by-side reviews. 80 is in fact too short.